README.md

```diff
@@ -61,7 +61,8 @@ new BundleAnalyzerPlugin(options?: object)
 |**`analyzerPort`**|`{Number}` or `auto`|Default: `8888`. Port that will be used in `server` mode to start HTTP server.|
 |**`reportFilename`**|`{String}`|Default: `report.html`. Path to bundle report file that will be generated in `static` mode. It can be either an absolute path or a path relative to a bundle output directory (which is output.path in webpack config).|
 |**`reportTitle`**|`{String\|function}`|Default: function that returns pretty printed current date and time. Content of the HTML `title` element; or a function of the form `() => string` that provides the content.|
-|**`defaultSizes`**|One of: `stat`, `parsed`, `gzip`|Default: `parsed`. Module sizes to show in report by default. [Size definitions](#size-definitions) section describes what these values mean.|
+|**`defaultSizes`**|One of: `stat`, `parsed`, `gzip`, `brotli`|Default: `parsed`. Module sizes to show in report by default. [Size definitions](#size-definitions) section describes what these values mean.|
```

Comment:
In this case user will have to provide compression algorithm twice: for `compressionAlgorithm` and `defaultSizes` and it opens possibilities for invalid option combinations like `defaultSizes: 'gzip', compressionAlgorithm: 'brotli'` which don't make sense. I personally would change `defaultSizes: 'gzip'` to `defaultSizes: 'compressed'` and deprecate the former, but support it (convert to `compressed`).

Reply:
The current implementation handles the mismatch gracefully, so it's more of a cosmetic issue on the user side.

Reintroducing `compressed` would help here. If used, the config always looks "good", even when switching the algorithm. _If_ we support `compressed`, fail on mismatches between `compressionAlgorithm` and `defaultSizes`?